ansible/accumulo.yml

```
@@ -53,26 +53,34 @@
       when: accumulo_major_version == '2' and not use_systemd
 - hosts: accumulomaster
   tasks:
-    - name: "start accumulo 2.0 master, monitor, gc & tracer"
+    - name: "start accumulo 2.x master, monitor, and gc"
       command: "nohup {{ accumulo_home }}/bin/accumulo-service {{ item }} start"
       register: start_result
       changed_when: "'Starting' in start_result.stdout"
       with_items:
         - master
         - monitor
         - gc
-        - tracer
       when: accumulo_major_version == '2' and not use_systemd
+    - name: "start accumulo 2.0 tracer"
+      command: "nohup {{ accumulo_home }}/bin/accumulo-service tracer start"
+      register: start_tracer_result
+      changed_when: "'Starting' in start_tracer_result.stdout"
+      when: accumulo_version is version('2.0.0','>=') and accumulo_version is version('2.1.0','<') and not use_systemd
 - hosts: accumulomaster
   tasks:
     - name: "install and start all the accumulo services using systemd"
       block:
         - import_tasks: roles/accumulo/tasks/start-master.yml
         - import_tasks: roles/accumulo/tasks/start-gc.yml
         - import_tasks: roles/accumulo/tasks/start-monitor.yml
-        - import_tasks: roles/accumulo/tasks/start-tracer.yml
       when: use_systemd
       become: yes
+    - name: "install and start all the accumulo tracer service using systemd"
+      block:
+        - import_tasks: roles/accumulo/tasks/start-tracer.yml
+      when: use_systemd and accumulo_version is version('2.1.0','<')
```

@keith-turner - sorry, I just spotted this, should it not be:
```suggestion
      when: accumulo_version is version('2.0.0','>=') and use_systemd and accumulo_version is version('2.1.0','<')
```
We really don't block someone from setting use_systemd to `True` when using Accumulo 1.x so I guess it is safer to filter this way.
to be consistent?